### PR TITLE
use hydration to make checklist look better before it completes loading

### PR DESCRIPTION
**Pull request by MichaelEbert**
Currently, checklist page is mostly blank before the js finishes creating the content.
This PR changes it to have all the elements already there, and the JS just hooks them up to the save file stuff behind the scenes.
This allows you to start scrolling down the checklist before everything is fully loaded.

### CODE REVIEW

File: checklist.html [hunk 1]
```diff
@@ -64,27 +64,29 @@
...
...g"><a href="https://en.uesp.net/wiki/Oblivion:Fort_Istirus" target="myframe" class="locationName">Fort Istirus</a><input type="checkbox" class="locationCheck check" id="location0x000174D0check"><a href="./map.html?formId=0x000174D0&amp;zoom=0.7" target="myframe" class="itemHelp" title="View on map">🗺️</a></div><div class="location item" id="location0x00017501"><img class="itemIcon"  draggable="false" src="images/Icon_Fort.png"><a href="https://en.uesp.net/wiki/Oblivion:Fort_Dirich" target="myframe" class="locationName">Fort Dirich</a><input type="checkbox" class="locationCheck check" id="location0x00017501check"><a href="./map.html?formId=0x00017501&amp;zoom=0.7" target="myframe" class="itemHelp" title="View on map">🗺️</a></div><div class="location item" id="location0x000174EC"><img class="itemIcon"  draggable="false" src="images/Icon_Fort.png"><a href="https://en.uesp.net/wiki/Oblivion:Fort_Rayles" target="myframe" class="locationName">Fort Rayles</a><input type="checkbox" class="locationCheck check" id="location0x000174ECcheck"><a href="./map.html?formId=0x000174EC&amp;zoom=0.7" target="myframe" class="itemHelp" title="View on map">🗺️</a></div><div class="location item" id="locat
```
Comment (MeemawHustlin) [inline checklist.html:84]: oh my lord

**Review by MeemawHustlin — COMMENTED** [review]
I'm getting the bricked settings issue again, just want to check before approving it if that's only my end or if you have the same bug with this PR on your device

File: checklist.html (hunk 1 above)
Comment (MichaelEbert) [reply checklist.html:84]: Yeah 😂 it's all the generated content just splorped into one line. We could manually linebreak it but we'd have to do that every time the content changes so it's not really worth it

**Comment by MichaelEbert** [comment]
oops. Thanks. my previous PR to fix it had another error.

**Comment by MichaelEbert** [comment]
Also, I split on divs because changing 1 thing and having git show the entire file as changed was getting old very fast

**Review by MeemawHustlin — APPROVED** [review]
Looks all good now

**Pull request merged**